libraries/AP_HAL_ChibiOS/hwdef/MFT-SEMA100/defaults.parm

```diff
@@ -0,0 +1,7 @@
+# setup the temperature compensation
+BRD_HEAT_TARG 45
+BRD_HEAT_P 50
+BRD_HEAT_I 0.07
+
+SERIAL3_BAUD 38
```

Comment:
see comment on last line of hwdef....not sure why these need to be set...as is, its already the default for GPS (which is autobauded anyway)

Reply:
Weird thing that when I check for the serials from MP the default Serial value was 230 for both Serıal 3 and 4 thats added to fix it. I can remove it if it is not necessary to show the default Baud as 38

Reply:
230 is the new default for GPS protocols....it was changed from 38 just reflect what the GPS driver actually uses....38 was not used...

Reply:
Alright now that makes sense =) Fixing it.